src/app/modals/property-management-modal/property-management-modal.component.html

```diff
@@ -0,0 +1,163 @@
+<div class="modal d-block">
+  <div class="modal-dialog modal-dialog-scrollable">
+    <div class="modal-content position-fixed top-0 end-0" style="width: 30rem; min-height: 100vh;"
+      id="modal-content-id">
+      <div class="modal-header">
+        <h1 class="modal-title fs-5" id="exampleModalLabel">Property</h1>
```

Comment:
Replace "exampleModalLabel" with something meaningful pls

Reply:
I removed the ID, I think the ID is no longer necessary